client/state/ui/guided-tours/selectors.js

```diff
@@ -30,10 +30,22 @@ export const getGuidedTourState = createSelector(
 		const tourState = getRawGuidedTourState( state );
 		const { shouldReallyShow, stepName = '' } = tourState;
 		const stepConfig = getToursConfig()[ stepName ] || false;
+
+		const shouldShow = (
+			// the Preview step requires a selected site
+			!! getSelectedSiteId( state ) &&
```

Comment:
Not all tours/steps will need this. I wonder if we could put this in the config, or infer it somehow.